yaml.go

```diff
@@ -17,7 +17,7 @@
 //
 // Source code and other details for the project are available at GitHub:
 //
-//	https://github.com/go-yaml/yaml
+//	https://github.com/yaml/go-yaml
```

Comment:
Please drop the commit that introduces this.

Reply:
Done.